docs/src/components/InteractiveExample/index.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
-
 import clsx from 'clsx';
 import { useCopyToClipboard } from 'usehooks-ts';
 
 import BrowserOnly from '@docusaurus/BrowserOnly';
 import CodeBlock from '@theme/CodeBlock';
 import AnimableIcon, { Animation } from '@site/src/components/AnimableIcon';
+import { useReduceMotion } from 'react-reduce-motion';
```

Comment:
We have own hook `useReduceMotion` 🫣
- https://github.com/software-mansion/react-native-reanimated/blob/main/src/reanimated2/hook/useReducedMotion.ts
- https://github.com/software-mansion/react-native-reanimated/blob/main/src/reanimated2/PlatformChecker.ts#L52-L59

Reply:
Let's use that

Reply:
Our `useReducedMotion` requires refreshing the page to detect accessibility changes. Is it ok?

Reply:
On the other hand the warning would be compatible with actual animation, so its fine